src/app/layout.tsx

```diff
@@ -31,11 +31,13 @@ export const metadata: Metadata = {
 
 export default function RootLayout({ children }: { children: React.ReactNode }) {
   return (
-    <html lang="en">
+    <html lang="en" className="dark">
       <body className={`${inter.variable} ${dmSans.variable} ${zenDots.variable}`}>
         <StoreProvider>
           <AppInit />
           <div className="layout">
+            <div className="main">Hello</div>
+            <h1 className="text-main text-[50px]">Hello</h1>
```

Comment:
https://tailwindcss.com/docs/font-size

usually text size if defined by `H` tags, so it's better to redefine the styles for these tags in `global.css` or just use other H tag